nautobot/dcim/models/devices.py

```diff
@@ -124,7 +154,7 @@ class DeviceType(PrimaryModel):
     ]
 
     class Meta:
-        ordering = ["manufacturer", "model"]
+        ordering = ["manufacturer", "model", "hardware_family"]
```

Comment:
Not sure this is needed - given that `manufacturer, model` is already a unique combination, the additional ordering doesn't seem like it would have any effect.